### PR TITLE
This was largely Unreadable

### PR DESCRIPTION
**Pull request by jbwill36**
This was largely unreadable and hard to follow. The translation was rough, with a lot of grammatical errors which would be offputting to many. This is a great concept, and deserves to get some attention.

### CODE REVIEW

**Comment by hnimminh** [comment]
thanks @jbwill36 for contribution, Look good,  let me take a when free-time!

**Comment by jbwill36** [comment]
I was really cleaning this up for you, because I would like to see this do something. Its a great idea but its hard to get people to back a security device if its not very well presented and the language is full of holes from translation. I am happy to help clean that presentation up, because I would like to see your project succeed.

**Comment by hnimminh** [comment]
Yes, I really appreciate your help. wrong terminology will make people confuse too. I will try to fix english on Readme soon. Thanks for your advices
